CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+0.38.1 (2022-02-25)
+-------------------
+- Bump `pipelinewise-tap-kafka` from `4.0.1` to `5.0.0`
+- Bump `pipelinewise-target-bigquery` from `1.1.1` to `1.2.0`
+- Prevent usage of extended transformation feature when FastSync exists
```

Comment:
we also bumped pipelinewise-transform-field to 2.3.0 with new transformations on proeprties of json objects